packages/frontend/.gitignore

```diff
@@ -0,0 +1,146 @@
+# Logs

```

Comment:
Currently frontend doesn't compile because the webpack config can't find the .gitignore file in the packages/frontend folder. Copied the root .gitignore to the frontend package

Reply:
Thanks ! We can leave it empty, I'll find a better fix later

Reply:
Hi, i've removed the content of the gitignore